test/events.html

```diff
@@ -52,6 +52,7 @@
       var apiUrlRaml;
 
       suiteSetup(function(done) {
+        this.timeout(120000);
```

Comment:
That is correct. IE 11 is slow and parsing RAML on this browser may take some time.
I'm not requesting a review on a element itself. This code base was already merged with the master. As per previous comment, I'm asking about visual side.